templates/opencodelists/user_activation_url.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class="row mt-5">
+  <div class="col-8 offset-1">
+
+    <div class="d-flex align-items-baseline">
+      <h2 class="mb-5 mr-3">{{ user.name }}</h2>
+      <small><i>{{ user.email }}</i></small>
+    </div>
+
+    <p>Send the link below to the new user so they can set their password:</p>
+
+    <pre>{{ url }}</pre>
+
```

Comment:
Maybe add a link for signed-in admins to add another user?

Reply:
Ooh, I like that!